internal/scorecard/plugins/plugin_runner.go

```diff
@@ -470,5 +481,40 @@ func runTests(csv *olmapiv1alpha1.ClusterServiceVersion, pluginType PluginType,
 	// clear name of operator deployment
 	deploymentName = ""
 
-	return suites, nil
+	return testResults, testResultsLog, nil
+}
+
+// applySelector apply label selectors removing tests that do not match
+func applySelector(tests []schelpers.Test, selector labels.Selector) []schelpers.Test {
+	for i := 0; i < len(tests); i++ {
+		t := tests[i]
+		if !selector.Matches(labels.Set(t.GetLabels())) {
+			// Remove the test
+			tests = append(tests[:i], tests[i+1:]...)
+			i--
+		}
+	}
+	return tests
+}
+
+// testResultToScorecardTestResult is a helper function for converting from the TestResult type
+// to the ScorecardTestResult type
+func testResultToScorecardTestResult(tr schelpers.TestResult) scapiv1alpha2.ScorecardTestResult {
```

Comment:
```suggestion
func newScorecardTestResultFromTestResult(tr schelpers.TestResult) scapiv1alpha2.ScorecardTestResult {
```
nit ^

Reply:
good suggestion, I want to do another refactor PR around this area and will consider this one in that PR.  thx.